src/NLog/Config/InstallationContext.cs

```diff
@@ -90,6 +91,12 @@ public InstallationContext(TextWriter logOutput)
         public bool IgnoreFailures { get; set; }
 
         /// <summary>
+        /// Whether installation exceptions should be rethrown. If IgnoreFailures is set to true,
+        /// this property has no effect (there are no exceptions to rethrow).
+        /// </summary>
+        public bool ThrowExceptions { get; set; }
```

Comment:
I've gone for `ThrowExceptions` to try and match the `LogManager.ThrowExceptions`. Although because of how this depends on `IgnoreFailures`, it might be better named `RethrowExceptions`?